lib/travis/listener.rb

```diff
@@ -15,7 +15,7 @@ def config
 
   module Listener
     class Config < Travis::Config
-      define  redis:   { url: 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
+      define  redis:   { url: ENV['LISTENER_REDIS_URL'] || 'redis://localhost:6379', namespace: 'sidekiq', network_timeout: 5 },
```

Comment:
I'm +0 to using the same env var across apps.  For, me the fact that a common env var like `REDIS_URL` or `DATABASE_URL` has a prefix tells me that it "belongs" to another app.  The specific case of `LOGS_DATABASE_URL` on the travis-logs apps is a leftover from a transitional time that I would like to change to `DATABASE_URL`.